src/query_sink.rs

```diff
@@ -197,21 +228,28 @@ com::class! {
     }
 }
 
+impl std::fmt::Debug for QuerySink {
```

Comment:
👌

Reply:
I removed this as this is no longer needed